docs/en/latest/control-api.md

```diff
@@ -38,6 +38,9 @@ apisix:
     port: 9090
 ```
 
+The control API server does not support parameter matching by default, if you want to enable parameter matching
```

Comment:
We should notify that the parameter matching only work with the API added by plugin, not the builtin one from APISIX.

Reply:
> 
I update the documention about control-api.md and indicate the config will efficetive with plugin's control API